package.json

```diff
@@ -11,9 +11,9 @@
   "license": "GPL-3.0",
   "dependencies": {
     "moment": "^2.22.2",
+    "mssql": "^5.0.3",
     "ramda": "^0.25.0",
     "tedious": "^2.3.1",
```

Comment:
mssql has it's own dependency on tedious 6.0.1 so this version of tedious is not needed.